staging/src/k8s.io/apimachinery/pkg/conversion/unstructured/converter_test.go

```diff
@@ -43,13 +43,13 @@ type B struct {
 
 type C struct {
 	A []A `json:"ca"`
-	B B `json:",inline"`
+	B `json:",inline"`
 	C string `json:"cc"`
 	D *int64 `json:"cd"`
-	E map[string]int `json:"ce"`
+	E map[string]int64 `json:"ce"`
 	F []bool `json:"cf"`
-	G []int `json"cg"`
-	H float32 `json:ch"`
+	G []int64 `json:"cg"`
+//	H float32 `json:"ch"`
```

Comment:
Because of some reason, floats translate differently in custom code and when using json marshaling - need to figure out why.